### PR TITLE
Improve what's new in 3.0 

### LINKED ISSUE #1752: misspelling, no mention of InMemory provider?
"ingore"-&gt;"ignore".
I was hoping to read about what (if anything) had been improved with the InMemory provider for EF Core 3.0? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 67b660a3-4b81-2f6e-fd19-a2c71e9f3b5b
* Version Independent ID: 5552bf61-9cfb-5152-0bc1-903cd4bd417d
* Content: [New features in EF Core 3.0 - EF Core](https://docs.microsoft.com/en-us/ef/core/what-is-new/ef-core-3.0/features)
* Content Source: [entity-framework/core/what-is-new/ef-core-3.0/features.md](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/what-is-new/ef-core-3.0/features.md)
* Product: **entity-framework**
* Technology: **entity-framework-core**
* GitHub Login: @divega
* Microsoft Alias: **divega**

### PR DESCRIPTION
**Pull request by divega**
Fixes #1752

### CODE REVIEW

**Comment by geppy** [comment]
Linking directly to "What's New in C# 8.0" could also be helpful ([suggested change](https://github.com/aspnet/EntityFramework.Docs/compare/divega-improve-whatsnew-3.0...geppy:link-to-whatsnew-in-csharp?expand=1)), if it's not too noisy.

**Comment by divega** [comment]
> Linking directly to "What's New in C# 8.0" could also be helpful ([suggested change](https://github.com/aspnet/EntityFramework.Docs/compare/divega-improve-whatsnew-3.0...geppy:link-to-whatsnew-in-csharp?expand=1)), if it's not too noisy.

Will do. Next time consider adding the change suggestion in an in-line comment using <kbd>control</kbd>+<kbd>g</kbd> so I can just merge it easily :smile:

**Comment by divega** [comment]
@aspnet/efteam can someone please review?

File: entity-framework/core/what-is-new/ef-core-3.0/features.md [hunk 1]
```diff
@@ -56,19 +60,86 @@ await foreach(var o in orders)
...
+  public string LastName { get; set; }
+  public string? MiddleName { get; set; }
+}
+```
+
+See [nullable reference types in the C# documentation](https://docs.microsoft.com/dotnet/csharp/whats-new/csharp-8#nullable-reference-types) for more details.
+
+## Interception of database operations
+
+The new interception API in EF Core 3.0 allows providing custom logic to bet invoked automatically whenever low-level database operations occur as part of the normal operation of EF Core. For example, when opening connections, commiting transactions, or executing commands.
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-3.0/features.md]:
```suggestion
The new interception API in EF Core 3.0 allows providing custom logic to be invoked automatically whenever low-level database operations occur as part of the normal operation of EF Core. For example, when opening connections, commiting transactions, or executing commands. 
```

File: entity-framework/core/what-is-new/ef-core-3.0/features.md [hunk 2]
```diff
@@ -56,19 +60,86 @@ await foreach(var o in orders)
...
+}
+```
+
+See [nullable reference types in the C# documentation](https://docs.microsoft.com/dotnet/csharp/whats-new/csharp-8#nullable-reference-types) for more details.
+
+## Interception of database operations
+
+The new interception API in EF Core 3.0 allows providing custom logic to bet invoked automatically whenever low-level database operations occur as part of the normal operation of EF Core. For example, when opening connections, commiting transactions, or executing commands. 
+
+Similarly to teh interception features that existed in EF 6, interceptors allow you to intercept operations before or after they happen. When you intercept them before they happen, you are allowed to by-pass execution and supply alternate results from the interception logic.
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-3.0/features.md]:
```suggestion
Similarly to the interception features that existed in EF 6, interceptors allow you to intercept operations before or after they happen. When you intercept them before they happen, you are allowed to by-pass execution and supply alternate results from the interception logic. 
```

File: entity-framework/core/what-is-new/ef-core-3.0/features.md [hunk 3]
```diff
@@ -56,19 +60,86 @@ await foreach(var o in orders)
...
+  {
+    // Manipulate the command text, etc. here...
+    command.CommandText += " OPTION (OPTIMIZE FOR UNKNOWN)";
+    return result;
+  }
+}
+``` 
 
-## Interception
+and register it with your `DbContext`:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-3.0/features.md]:
```suggestion
And register it with your `DbContext`: 
```

File: entity-framework/core/what-is-new/ef-core-3.0/features.md [hunk 4]
```diff
@@ -56,19 +60,86 @@ await foreach(var o in orders)
...
 Entity types without keys (previously known as [query types](xref:core/modeling/keyless-entity-types)) represent data that can be read from the database, but cannot be updated.
 This characteristic makes them an excellent fit for mapping database views in most scenarios, so we automated the creation of entity types without keys when reverse engineering database views.
 
+For example, using the [dotnet ef command-line tool](xref:core/miscellaneous/cli/dotnet) you can type:
+
+``` console
+dotnet ef dbcontext scaffold "Server=(localdb)\mssqllocaldb;Database=Blogging;Trusted_Connection=True;" Microsoft.EntityFrameworkCore.SqlServer
+```
+
+And the tool will now automaticall scaffold types for views and tables without keys:
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-3.0/features.md]:
```suggestion
And the tool will now automatically scaffold types for views and tables without keys:
```

**Comment by geppy** [comment]
> Next time consider adding the change suggestion in an in-line comment using control+g so I can merge it more easily 😄

Will do!  Thanks for the comment, I didn't know how to do that.

**Pull request merged**